driver/docker/docker.go

```diff
@@ -153,11 +158,41 @@ func (d *Driver) exec(op *driver.Operation) error {
 		AttachStdout: true,
 	}
 
-	hostCfg := &container.HostConfig{AutoRemove: true}
+	outputsFolder, err := ioutil.TempDir(d.config["OUTPUTS_MOUNT_PATH"], "outputs")
+	if err != nil {
+		return driver.OperationResult{}, err
+	}
+	defer os.RemoveAll(outputsFolder)
+
+	// On Mac, the default temp folder (set by the $TMPDIR env variable) is in /var.
```

Comment:
Thanks for spotting that, just added this feedback to the Docker Desktop Team, should be shortly fixed 👍